blocks/filters/filters.js

```diff
@@ -1,7 +1,8 @@
-const ignoredFacets = ['assetID', 'tags', 'multiple', 'background', 'categories', 'foreground', 'sourceDomain'];
+const ignoredFacets = ['assetID', 'tags', 'multiple', 'background', 'categories', 'foreground'];
+const ignoreSourceDomains = ['cc-author.prod.corp.adobe.com', 'marketinghub', 'stock.adobe.com', 'www.hp.com', 'photoshopcamera.app.link', 'adobe-robohelp-launch-2020.meetus.adobeevents.com', 'main--blog--adobe.hlx.page', 'blogs.nvidia.com', 'www.adobeprerelease.com'];
```

Comment:
I would also ignore `fonts.adobe.com`. This was not really intended to be indexed (I think this is garbage data in the current index).

In fact, we only really have actively sourced `blog.adobe.com` and `www.adobe.com` and thus instead of an _exclusion_ list, this could be a much shorter _inclusion_ list for now.

(BTW, showing less facets probably leads to the count not adding up, but I guess we don't care right now and probably also hard to notice as a user)

For reference, current view when searching all assets:

<img width="260" src="https://user-images.githubusercontent.com/229022/170779437-a0f475e7-654f-4c4c-902f-204ee92be006.png">

Reply:
Done as of https://github.com/adobe/helix-assets-ui/pull/11/commits/b136ea43d82b0078fd2d6a9c3f097d570d0e3839.